bsip-0047.md

```diff
@@ -131,42 +131,36 @@ _General validation checks_
 * Validate the existence of each vote to add and remove
 
 _Committee validation checks_
-* Let the new _committee_ slate consist of the current _commitee_ slate plus the _committee_ `votes_to_add` minus the _committee_ `votes_to_remove`
-* If `account_options.committee_voting_account == GRAPHENE_PROXY_TO_SELF_ACCOUNT`
-	* Let `new_committee_account` = `GRAPHENE_PROXY_TO_SELF_ACCOUNT`
+* Let the new committee slate consist of the current commitee slate plus the committee `votes_to_add` minus the committee `votes_to_remove`
+* If `committee_voting_account` is present
+	* Let `new_committee_account` = `committee_voting_account`
 * otherwise
 	* Let `new_committee_account` = `account_options.committee_account`
 * If `num_committee` is present
 	* Let `new_num_committee` = `num_committee`
 * otherwise
 	* Let `new_num_committee` = `account_options.num_committee`
-* `num_committee`, if present, is valid:
+* The _committee_ `votes_to_add`,`votes_to_remove`, and `num_committee`, if present, are valid:
```

Comment:
I suppose you mean "valid if present" in the sense of "only allowed when ..."?

Reply:
I agree that "only allowed" is clearer.  I have updated the text to reflect that `num_X` and `votes_to_add` and `votes_to_remove` have their own separate validations in an attempt to be clear to the reader especially because `num_X` _may or may not_ be present, and `votes_to_add` and `votes_to_remove` _may or may not_ be non-empty.

The entire validation code can be written in a more computational efficient manner to reflect this intent.